plugins/pack-deploy.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: pack-deploy
+spec:
+  version: "v0.1.0"
+  platforms:
+  - uri: https://github.com/dougsland/kubectl-pack-deploy/archive/kubectl-pack-deploy-0.1.0.tar.gz
+    sha256: 403b04620f47c8d67f09c0aa405723fc9acdf3cc98face60f4adb9ff8e358d7e
+    bin: kubectl-pack_deploy
+    files:
+    - from: "./*/kubectl-pack_deploy"
+      to: .
+    - from: "./*/LICENSE"
+      to: .
+    selector:
+      matchExpressions:
+      -   {key: os, operator: In, values: [darwin, linux]}
+  caveats: |
+    This plugin needs the following programs:
+    * kubectl neat plugin
+    Examples:
+    To generate the YAML file from a kibana deployment running in the cluster use:
+      kubectl pack-deploy kibana
+  description: |
+    Detects resources required from running deployments, statefulsets,
+    daemonsets and export it with all dependencies in a single YAML.
+  shortDescription: Export running deployments, statefulsets, daemonsets with all dependencies in a single YAML
```

Comment:
Better stay below ~50chars, everything above will get truncated.

Also, this description is a bit of a false promise. Try to capture the gist of your logic like

> Export all resources with a matching name/label

Reply:
Sure, I will reduce to below ~50 chars. 

> Also, this description is a bit of a false promise. Try to capture the gist of your logic like
Not sure if I follow that. gist logic ?

Reply:
Gist: in the sense of "essential bits"